sample-app/src/main/kotlin/com/sats/dna/sample/components/PlaceholderScreen.kt

```diff
@@ -0,0 +1,54 @@
+package com.sats.dna.sample.components
+
+import androidx.compose.foundation.layout.Arrangement.spacedBy
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment.Companion.CenterVertically
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.unit.dp
+import com.sats.dna.components.PlaceholderBox
+import com.sats.dna.components.PlaceholderParagraph
+import com.sats.dna.components.PlaceholderText
+import com.sats.dna.theme.SatsTheme
+import com.sats.dna.tooling.LightDarkPreview
+
+@Composable
+internal fun PlaceholderScreen(navigateUp: () -> Unit) {
+    ComponentScreen("Placeholders", navigateUp) { innerPadding ->
+        Column(Modifier.padding(innerPadding)) {
+            PlaceholderBox(
+                Modifier
+                    .aspectRatio(16f / 9)
+                    .fillMaxWidth(),
+                RectangleShape,
+            )
+
+            Column(Modifier.padding(SatsTheme.spacing.m), spacedBy(SatsTheme.spacing.m)) {
```

Comment:
Okay super nitpicky here, but does the verticalArrangement deserve a named parameter here? Makes it a lot easier to read 🧐 
Also mby a new line to make it more readable as well, but idk if we have any conventions on it? Idk if I just come from a very strict code style world, but in my head composables should very rarely have single line argument "lists"🤔

Reply:
“Super nitpicky” is what I live for! :tada: 

And I have nothing against introducing a named parameter on a new line! In the context of a `Column`, my head automatically thinks “spacing around items must be vertically arranged`, but I see that's not necessarily obvious. :+1: